packages/flutter_tools/test/general.shard/build_system/targets/asset_transformer_test.dart

```diff
@@ -0,0 +1,377 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:args/args.dart';
+import 'package:file/memory.dart';
+import 'package:file_testing/file_testing.dart';
+import 'package:flutter_tools/src/artifacts.dart';
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/logger.dart';
+import 'package:flutter_tools/src/build_system/tools/asset_transformer.dart';
+import 'package:flutter_tools/src/flutter_manifest.dart';
+
+import '../../../src/common.dart';
+import '../../../src/fake_process_manager.dart';
+
+void main() {
+  test('Invokes dart properly', () async {
+    final FileSystem fileSystem = MemoryFileSystem();
```

Comment:
nit. `MemoryFileSystem.test()` (which uses a fake clock)

Reply:
Addressed in [nit](https://github.com/flutter/flutter/pull/143815/commits/87d31c3dbc1bdbc1630ea9fdfb6636380a67d8f7)